sci-misc/boinc/files/boinc.init.in

```diff
@@ -15,13 +15,13 @@ create_work_directory() {
 	if [ ! -d "${RUNTIMEDIR}" ]; then
```

Comment:
But you're not bumping the version in this commit which makes the commit message wrong and confusing. Focus summary line on telling what you're fixed exactly, you've got enough space there to put shortly what exact issue is fixed. 

Also, the footer tag is `Bug:`, not `See:`. Please also remove the empty line between it and `Package-Manager`.